### ISSUE TITLE
RC support needs to be added back in before DKPY 2 is released

### ISSUE THREAD

**Issue by hamishwillee** [dronekit/dronekit-python#358]
channel_override support was removed during beta test, but is to be added before we create an official release. This issue has been added as a reminder.

This was reported by a user on the gitter channel:

Found an error in DK2.0.0b6
```
fw.channel_override = { "1" : 1500 , "2" : 1500, "3" : 1250, "4" : 1500 }
File "/usr/local/lib/python2.7/dist-packages/dronekit/module/api.py", line 238, in channel_override
overrides = self.rc.override
AttributeError: 'MPVehicle' object has no attribute '_MPVehiclerc'
```

**Comment by tcr3dr** [comment]
See #427.

**Comment by tcr3dr** [comment]
Implemented as of #427.
